From 41f2d2ce4635c8a902445bc2df66d14ee4fa81eb Mon Sep 17 00:00:00 2001
From: Hiroshi Hatake <hatake@clear-code.com>
Date: Tue, 11 Apr 2017 13:38:17 +0900
Subject: [PATCH 44/44] Fix build error for invalid symbol

```
/usr/bin/ld: libxul.so: hidden symbol
`_ZNK14gfxPlatformGtk21SupportsApzTouchInputEv' is not defined.
```
---
 gfx/thebes/gfxPlatformGtk.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/gfx/thebes/gfxPlatformGtk.h b/gfx/thebes/gfxPlatformGtk.h
index 982390d1867b..7707060dddad 100644
--- a/gfx/thebes/gfxPlatformGtk.h
+++ b/gfx/thebes/gfxPlatformGtk.h
@@ -121,7 +121,9 @@ public:
       return true;
     }
 
+#ifdef GL_PROVIDER_GLX
     bool SupportsApzTouchInput() const override;
+#endif
 
     void FontsPrefsChanged(const char *aPref) override;
 
-- 
2.11.0

